### PR TITLE
Updated README with more relevant information

### PR DESCRIPTION
**Pull request by IgnacioPro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,70 @@
...
+| `months`   | Download files from the past x months. | `int` | `0` | No        |
+| `years`   | Download files from the past x years. | `int` | `0` | No        |
 
 
-## Additional Documentation and Acknowledgments
 
-* CRE Team @ Red Hat
+## ⚠️ License
 
+`Syncron` is free and open-source software licensed under the [Apache 2.0 License](https://github.com/create-go-app/cli/blob/master/LICENSE).
```
Comment (sdatko) [inline README.md]: better link to syncron repo maybe?

Comment (IgnacioPro) [reply README.md]: Yes, that would be preferable :D

File: README.md [hunk 2]
```diff
@@ -1,56 +1,70 @@
-# Syncron
+<h1 align="center">
+  <img alt="cgapp logo" src="https://seeklogo.com/images/G/go-logo-046185B647-seeklogo.com.png" width="100px"/><br/>
+  Syncron
+</h1>
+<p align="center">Easily fetch files from  <b>S3 buckets</b> with a cli based application written in Golang.</p>
 
-Golang CLI project to manage s3 buckets from AWS.
+<p align="center"><a href="https://pkg.go.dev/github.com/create-go-app/cli/v3?tab=doc" 
+target="_blank"><img src="https://img.shields.io/badge/Go-1.18+-00ADD8?style=for-the-badge&logo=go" alt="go version" /></a>&nbsp;<a href="https://gocover.io/github.com/create-go-app/cli/pkg/cgapp" target="_blank"><a href="https://goreportcard.com/badge/github.com/redhatcre/syncron" target="_blank"><img src="https://img.shields.io/badge/Go_report-A+-success?style=for-the-badge&logo=none" alt="go report" /></a>&nbsp;<img src="https://img.shields.io/badge/license-apache_2.0-red?style=for-the-badge&logo=none" alt="license" /></p>
```
Comment (sdatko) [inline README.md]: I would avoid mixing HTML and Markdown unless it is strictly necessary.

For the badges, you do not add any styling, so replacing `<a>` and `` with `[name](link to your URL)` would be better IMO.

**Review by sdatko — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sdatko) [reply README.md]: How about:
```
<div style="text-align: center;">
  <img alt="cgapp logo" src="https://seeklogo.com/images/G/go-logo-046185B647-seeklogo.com.png" width="100px"/>

# Syncron

Easily fetch files from **S3 buckets** with a cli based application written in Golang.

[![GoProject](https://img.shields.io/badge/Go-1.18+-00ADD8?style=for-the-badge&logo=go)](https://pkg.go.dev/github.com/create-go-app/cli/v3?tab=doc)
[![GoReport](https://img.shields.io/badge/Go_report-A+-success?style=for-the-badge&logo=none)](https://goreportcard.com/badge/github.com/redhatcre/syncron)
![License](https://img.shields.io/badge/license-apache_2.0-red?style=for-the-badge&logo=none)
</div>
```

File: README.md [hunk 3]
```diff
@@ -1,56 +1,70 @@
...
 
-Golang CLI project to manage s3 buckets from AWS.
+# Syncron
 
+Easily fetch files from **S3 buckets** with a cli based application written in Golang.
 
-## Getting Started
+[![GoProject](https://img.shields.io/badge/Go-1.18+-00ADD8?style=for-the-badge&logo=go)](https://github.com/RedHatCRE/syncron) [![GoReport](https://img.shields.io/badge/Go_report-A+-success?style=for-the-badge&logo=none)](https://goreportcard.com/badge/github.com/redhatcre/syncron) ![License](https://img.shields.io/badge/license-apache_2.0-red?style=for-the-badge&logo=none)
+</div>
+## ⚡️ Quick start
```
Comment (sdatko) [inline README.md:11]: Empty line required above `##` to make it effective.

File: README.md [hunk 4]
```diff
@@ -1,56 +1,70 @@
-# Syncron
+<div style="text-align: center;">
```
Comment (sdatko) [inline README.md]: It appears GitHub removes the style attribute... please change it back to `align="center"` instead :-)

**Review by adrianfusco — APPROVED** [review]
After the changes requested by Szymon +1

**Review by sdatko — APPROVED** [review]

**Comment by sdatko** [comment]
Now it is perfect

**Pull request merged**